force-app/main/default/lwc/bulkServiceDeliveryUI/bulkServiceDeliveryUI.js

```diff
@@ -211,7 +223,10 @@ export default class BulkServiceDeliveryUI extends NavigationMixin(LightningElem
     }
 
     get isDoneDisabled() {
-        return Object.keys(this.errors).length > 0;
+        if (Object.keys(this.errors).length > 0) {
```

Comment:
Since we are modifying the isDoneButtonDisabled variable which is not on the markup, the done button will not actually get disabled. We should be able to combine the clearError and autoSave complete since they are both called onSuccess of the row being saved. Please let me know if you would like to hop on a call to discuss. Thanks!

Reply:
@jjbennett , Totally agree that the isDoneButtonDisabled is not used in the markup but isDoneDisabled is used and since this returns either a true or false it disables or enables the button. Please let me know if that is fine else we can hop on a call.